### DISCUSSION TITLE
System Tray / Notification Area Icons

### DISCUSSION THREAD

**Discussion by Tularis1** [valinet/ExplorerPatcher#578] [Q&A]
Hello,

Is there a way to get all of my System Tray Icons (Notification Area Icons) to appear on the Task Bar on my Second Monitor?
I can get the Clock and such but apps like Outlook, Onenote Icons are only on the 1st Monitor?

**Comment by valinet** [comment]
Yeah, that's the functionality that's currently built into Windows. Replicating the system tray icons on other taskbars is actually not that easy programatically-wise, due to how the system tray is designed so that's why Microsoft probably never bothered to implement it officially. Although, I heard there are third party programs that did it, like Actual Multiple Monitors (?). But yeah, I don't have experience with any of those, I never sought such functionality.

**Reply by S1332** [reply]
It was possible to pull the taskbar from the main screen to the secondary so everything was over there without fucking up the primary screen setting for fullscreen games
